chain/epoch_manager/src/lib.rs

```diff
@@ -465,6 +466,17 @@ impl EpochManager {
         }
         Ok((stake_info, validator_reward))
     }
+
+    /// Get validators for current epoch and next epoch.
+    pub fn get_validator_info(
+        &mut self,
+        block_hash: &CryptoHash,
+    ) -> Result<EpochValidatorInfo, EpochError> {
+        let current_validators = self.get_epoch_info_from_hash(block_hash)?.validators.clone();
+        let next_epoch_id = self.get_next_epoch_id(block_hash)?;
+        let next_validators = self.get_epoch_info(&next_epoch_id)?.validators.clone();
+        Ok(EpochValidatorInfo { current_validators, next_validators })
```

Comment:
let's add proposals for current proposals / kickouts by calling `collect_blocks_info`